packages/OmegaPrint-Core.package/OPPrinter.class/instance/MoreExecutableCode.with.and..st

```diff
@@ -0,0 +1,11 @@
+actions
+MoreExecutableCode: aNode with: statements and: aFinalStatement
+
+	| result |
+	result := self value: statements.
+	aFinalStatement children first
+		ifNotNil: [
```

Comment:
Bin mir nicht sicher, ob das `ifNotNil` eine eigene Zeile braucht. Andererseits kommt man dadurch der rechteckigen Block-Form naeher. Was meinen die anderen?

Reply:
ich würde es in eine extra Zeile schreiben, weil ich das übersichtlicher finde woran das geschickt wird

Reply:
Ich würde Felix hier recht geben, haben das auch in der User Story so geschrieben

Reply:
Du wurdest leider von Tarik und mir demokratisch ueberstimmt